tools/webidl_binder.py

```diff
@@ -629,7 +629,7 @@ def make_call_args(i):
     if non_pointer:
       return_prefix += '&'
     if copy:
-      pre += '  static %s temp;\n' % type_to_c(return_type, non_pointing=True)
+      pre += '  static thread_local %s temp;\n' % type_to_c(return_type, non_pointing=True)
```

Comment:
Please add a comment, maybe something like "Avoid sharing this static temp var between threads, which could race." Otherwise looks great.

Reply:
Done.